package-manifest.toml

```diff
@@ -123,7 +123,7 @@ source.type = "local"
 source.rust.binary_names = ["oximeter", "clickhouse-schema-updater"]
 source.rust.release = true
 source.paths = [
-    { from = "smf/oximeter", to = "/var/svc/manifest/site/oximeter" },
+    { from = "smf/oximeter/manifest.xml", to = "/var/svc/manifest/site/oximeter/manifest.xml" },
```

Comment:
The `smf/oximeter` directory contains a `config.toml` file -- with this change, it looks like that's no longer included in the package.

If we aren't using that config file, should we remove it from the git repo?

Reply:
reverted, not sure what I was going for here